packages/next-tinacms-cloudinary/src/options.ts

```diff
@@ -0,0 +1 @@
+export type TinaCloudCloudinaryOptions = { baseURL: string }
```

Comment:
`baseURL` -> `baseUrl`

```suggestion
export type TinaCloudCloudinaryOptions = { baseUrl: string }
```